README.md

```diff
@@ -1 +1,44 @@
-# ababaug.github.io
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# I designed the desktop version of the milestone portfolio project.
+
+> The project is to create the mobile version of my Portfolio.
+
+![Screenshot](./images/shot.png)
+
```

Comment:
- [ ] Kindly add a link to  the live demo of your deployed project to your readme 👍🏿